lib/conjur/api/resources.rb

```diff
@@ -166,5 +166,19 @@ def resources opts = {}
     def global_privilege_permitted? privilege
       resource(GLOBAL_PRIVILEGE_RESOURCE).permitted? privilege
     end
+
+    def resources_permitted? kind, identifiers, privilege
+      options = {
+        privilege: privilege,
+        identifiers: identifiers
```

Comment:
In the authz changeset, it looks like this parameter should be called `resource_ids` but that's a bit misleading (in authz), because authz is not actually expecting them to be resource ids, its expecting them to be only resource identifiers, with `account` and `kind` supplied by the route parameters.